### ISSUE TITLE
mnist_dpsgd_tutorial_keras.py getting accuracies of about 10%

### ISSUE THREAD

**Issue by weiyangwang** [tensorflow/privacy#58]
When running mnist_dpsgd_tutorial_keras.py the accuracies are only about 10%, however the standard model without DPGradientDescentGaussianOptimizer gets close to 100% accuracy. mnist_dpsgd_tutorial.py with DP however gives close to 95% accuracy.

 
<img width="564" alt="Screen Shot 2019-07-24 at 11 46 52 AM" src="https://user-images.githubusercontent.com/7920628/61763711-ee26c380-ae08-11e9-92c8-7389bdb3f760.png">

**Comment by npapernot** [comment]
Can you describe your setup? OS, TF version, etc...

**Comment by weiyangwang** [comment]
I've tried both tf 1.14.0 and 2.0.0-beta1, and on both MacOS High Sierra and colab notebook. Both gives same results.

**Comment by npapernot** [comment]
I just ran the tutorial on a TF14 fresh install and couldn't reproduce the bug. Did you make any modifications to the tutorial?

**Comment by weiyangwang** [comment]
You're right. I retried and verified that it works well with TF14. I was experimenting with TF 2.0.0-beta1 and could run the code after changing the logging to abseil and switching out the tf.contrib dependencies. But this gives only about 10% accuracy. 

So the issue is with using TF 2.0.0-beta1.

**Comment by npapernot** [comment]
I'm closing for now since we are still building up support for TF2. If you have specific bugs you identify with TF2, feel free to open separate issues for them.
